pretty.c

```diff
@@ -316,6 +316,20 @@ static void print_in_formats(struct json_object *arr, const char *prefix)
 	}
 }
 
+static void print_hdr_output_metadata(struct json_object *arr, const char *prefix)
+{
+	struct json_object_iter iter;
+	json_object_object_foreachC(arr, iter) {
+		if (iter.val) {
+			printf("%s%s\"%s\" = %"PRIu64"\n",
```

Comment:
I'd prefer to manually print each field here. This would allow us to get human-readable output. Enums could be pretty-printed with the stringified enum name and its value (like we do for modifiers, but no need for code generation). Fields with X/Y values could be printed on a single line.